### ISSUE TITLE
Bug: After opening the Files main window it is blocked for several minutes before it can be used

### ISSUE THREAD

**Issue by manfredu** [files-community/Files#16552]
### Description

Even if Files is started together with windows it takes approx. 10 minutes after opening the Files main window until it can be used. 

During this time it is completely blocked. Afterwards Files is working fine

Possible reasons (just guessing):

- Network drives are not visible on the left under "Network"
- OneDrive is used
- Network detection and file sharing is disabled

### Steps To Reproduce

1. Start Files
2. Window appears, but it is completely blocked
3. Wait for several minutes, use other programs in the meantime, e.g. Windows Explorer ;-)
4. If you wait long enough, Files can be used without issues.

### Files Version

3.8.0.0

### Windows Version

Win 11 Enterprise 23H2 22631.4460

### Log File

Did not find the log. May be hidden by company policies.

**Comment by manfredu** [comment]
Just used Files a little while, now it's window is greyed out again and the cursor shown as hourglass when moving over the Files window... Last action I did was selecting the videos folder (Which is empty).

Files is unusable for me as it looks.

**Comment by Josh65-2201** [comment]
Thanks for the report, 
- Do you have network drives connect and are they set to offline/disconnected. 
- Do you have `Files settings > Files & Folders > Calculate folder sizes` on? If so try with it off.
- Can you try turning off `Files settings > Advanced > Leave app running in the background when the window is closed`

**Comment by manfredu** [comment]
> Thanks for the report,
> 
>     * Do you have network drives connect and are they set to offline/disconnected.

No.
  
> 
>     * Do you have `Files settings > Files & Folders > Calculate folder sizes` on? If so try with it off.

No.

> 
>     * Can you try turning off `Files settings > Advanced > Leave app running in the background when the window is closed`

Ok, I've changed this setting to off. It looks like these extrem delays are no longer occuring, but I will watch it and report back.

**Comment by yaira2** [comment]
Do you have a lot of items in Recycle Bin?

**Comment by manfredu** [comment]
Yes, it contains 105623 files.

I was not aware of this. Opened it now with Windows Explorer, which counted the files for a while. But did not block the ui in the meantime.

**Comment by yaira2** [comment]
Does this issue go away if you empty Recycle Bin? If it does, I believe you've come across this issue #14108.

**Comment by manfredu** [comment]
> Does this issue go away if you empty Recycle Bin? If it does, I believe you've come across this issue [#14108](https://github.com/files-community/Files/issues/14108).

Yes indeed, emptiing the recycle bin made all the delays go away. :-)

In my opinion this is something which needs to be solved soon because everybody who does not know about this issue just thinks that Files is a slow application... ;-). And deleting files is nothing special and moving them to the recycle bin is the default I believe. And it's not obvious that this is the reason for the unresponsive behaviour.

**Comment by yaira2** [comment]
I tried fixing it in the past but didn't have any success, however, perhaps someone else will have better luck. Anyways, let's continue tracking in #14108.
